bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -12,4 +12,56 @@ components:
       test-configs:
         - with-security
         - without-security
+  - name: anomaly-detection
+    integ-test:
+      dependencies:
+        - job-scheduler
+        - alerting
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      dependencies:
+      test-configs:
+        - with-security
+        - without-security
+  - name: asynchronous-search
+    integ-test:
+      dependencies:
+        - job-scheduler
+        - alerting
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      dependencies:
+      test-configs:
+        - with-security
+        - without-security
+  - name: sql
+    integ-test:
+      dependencies:
+        - job-scheduler
+        - alerting
```

Comment:
(outside of scope of this PR) we should remove these `dependencies` in a separate PR and remove them from test_manifest schema. This will not be used.

Reply:
Removed the `dependencies` and removed from schema as well